cucumber/authenticators_jwt/features/authn_jwt_fetch_identity_from_url.feature

```diff
@@ -0,0 +1,49 @@
+Feature: JWT Authenticator - Fetch identity from URL
+  Background:
+    Given I initialize JWKS endpoint with file "myJWKs.json"
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: jwks-uri
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !host myapp
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+
+  Scenario: Host taken from URL but not defined in conjur, error
+    Given I issue a JWT token:
+    """
+    {
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with invalid_host account in url
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    Errors::Authentication::Security::RoleNotFound: CONJ00007E
```

Comment:
what about the message in logs ?

Reply:
The error message is 
Errors::Authentication::Security::RoleNotFound: CONJ00007E 'invalid_host' not found

But the value can change according to content of test so i put the first part that is not changing

Reply:
On this test it should be the same all the time, its important to add it

Reply:
thats not enough, the log message on each test is always the same depends on the test scenario, each time you run this test the log message will be the same, therefor you should write the expected log message
and please remove the `Errors::Authentication::Security::RoleNotFound:` part

Reply:
Fixed